code/game/turfs/open/_open.dm

```diff
@@ -1,5 +1,7 @@
 /turf/open
 	plane = FLOOR_PLANE
+
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 20, ACID = 0)
```

Comment:
The cost of this is going to be *enormous*, I need to look into this

Reply:
I believe getArmor takes a type path in place of this

Reply:
i could remove this and just increase the integrity by 20%